Mobile/starter_project_matador/lib/features/example/presentation/screen/signup_page.dart

```diff
@@ -0,0 +1,49 @@
+// Import flutter heplper library
+import 'package:flutter/material.dart';
+import '../widgets/logo.dart';
+import '../widgets/signup_and_signin_box.dart';
+import '../widgets/data_fill.dart';
+import '../widgets/white_box.dart';
+
+class SignUpPage extends StatelessWidget {
+  const SignUpPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        backgroundColor: Colors.white,
```

Comment:
Avoid giving colors this way. There is a file where colors are defined inside the core/utils/constants directory. use that file to define colors and use them. This apply for all colors.

Reply:
done avoiding it